scripts/cleanup_failed_registrations.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TEMPDIR=`mktemp -d`
+trap "rm -rf $TEMPDIR" EXIT
```

Comment:
This instructs the system to create a random temporary directory, when the script exits for reasons good or bad the `trap` ensures the temporary directory is removed.